### ISSUE TITLE
Less: passing ruleset with comment  to mixin call duplicates the closing semicolon of mixin call

### ISSUE THREAD

**Issue by slorber** [prettier/prettier#3096]
**Prettier 1.7.4**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAdKA6AtgTwLICWAHgVABQACAzhFnISVAEoCuANnFUgATDrfcA9IO6QsdWP24AvALSkAJnCI8ADAG50AXwCUmqOhAAaEBAAOMAtCrJQAQyoxkAMztsqcEwCMATnbAA1nAwAMpm-qQA5sgwPiyeIAoQYC5uHiYAVlREAEJ+gcEhdnQAMqRwqe4JECwwZrUATJXpIOE+Hj7IIBxUNiZmPqQwAOoECjAAFsgAHKomHlgEMXEJVFEcAIosEPDNCTB2XqPjU0gNJrF2BGxRAMK0WHZdbmzGICweACqHNkiuVVotEA)
```sh
--parser less
--trailing-comma all
```

**Input:**
```less
.myMixin(@someMixinRules: {
  // comment
  z-index: 0;
});
```

**Output:**
```less
.myMixin(@someMixinRules: {
  // comment
  z-index: 0;
});;
```

**Expected behavior:**

```less
.myMixin(@someMixinRules: {
  // comment
  z-index: 0;
});
```

**Comment by slorber** [comment]
It does not cause compilation errors, and it's easy to move/delete the comment as a workaround

**Comment by thorn0** [comment]
While there are still [problems with formatting detached rulesets](https://github.com/prettier/prettier/issues/4524), this specific issue with semicolons is gone.

**Comment by thorn0** [comment]
This is going to be fixed by #6981.

**Comment by thorn0** [comment]
fixed by #6981
